oauth_client.go

```diff
@@ -95,6 +95,8 @@ type OAuthClient struct {
 	// Relations
 	Organization *Organization `jsonapi:"relation,organization"`
 	OAuthTokens  []*OAuthToken `jsonapi:"relation,oauth-tokens"`
+	AgentPool    *AgentPool    `jsonapi:"relation,agent-pool"`
```

Comment:
It appears that you've also added the inverse relationship to the API (each agent pool has_many oauth-clients). What do you think about adding `include_data = false` to that new relationship in the API? Is there a good reason to serialize all of the oauth-clients associated with an agent pool?